guard/tests/utils.rs

```diff
@@ -64,6 +65,37 @@ pub fn get_full_path_for_resource_file(path: &str) -> String {
     return resource.display().to_string();
 }
 
+pub fn replace_home_directory_with_tilde(text: String) -> String {
+    let home_dir_string = match env::var("HOME") {
+        Ok(home_path) => home_path,
+        Err(_) => panic!("HOME variable required for tests!"),
+    };
+
+    text.replace(&home_dir_string, "~")
+}
+
+pub fn replace_path_with_filenames(text: String) -> String {
+    let extensions = ["yaml", "yml", "json"];
+    // pattern to match anything between "~/" and any of the extensions
+    let pattern = format!(
+        r#"~/(?:[\w/\-]+/)?([\w/\-]+\.(?:{}))"#,
+        extensions.join("|")
+    );
+    let re = Regex::new(&pattern).unwrap();
```

Comment:
Full transparency I have no problem calling unwrap() here without attempting to handle the error, since this is a code path that only gets executed during our tests.

Reply:
Just went with this since it's only used in test but can handle differently with a message if you think thats better